src/components/ChatHeader/ChatHeader.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { IconButton } from '@mui/material';
+import PropTypes from 'prop-types';
+import CallIcon from '@mui/icons-material/Call';
+import ProfileBadge from '../ProfileBadge/ProfileBadge';
+import styles from './ChatHeader.module.css';
+
+/**
+ * ChatHeader component displays the avatar, name and a status of the buddy the current
+ * user in the current conversation.
+ * A call button will allow user to give a call to this buddy.
+ */
+function ChatHeader({ active, name, callBuddy }) {
+  return (
+    <div className={styles.chatHeader}>
```

Comment:
You can try use more mui components next time. It will save u lots time from styling.